src/OpenSpaceToolkit/Astrodynamics/EventCondition/AngularCondition.cpp

```diff
@@ -67,7 +73,7 @@ void AngularCondition::print(std::ostream& anOutputStream, bool displayDecorator
 
     EventCondition::print(anOutputStream, false);
     ostk::core::utils::Print::Line(anOutputStream) << "Criterion: " << StringFromCriterion(getCriterion());
-    if (target_.isDefined())
+    if (target_.value_.isDefined())
```

Comment:
nit: could possibly also be worth it to make a `Target::isDefined()` that checks the value and relative state properly based on the target type.